CHANGELOG.md

```diff
@@ -2,8 +2,16 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
-## [1.13.0](https://github.com/aws-observability/aws-rum-web/compare/v1.12.0...v1.13.0) (2023-02-23)
+## [1.14.0](https://github.com/aws-observability/aws-rum-web/compare/v1.13.0...v1.14.0) (2023-06-29)
+
+### Features
+
+* Filter unwanted performance events with custom callback. By default, ignore resources with non-http url scheme. ([#419](https://github.com/aws-observability/aws-rum-web/issues/419)) ([42a2ae5](https://github.com/aws-observability/aws-rum-web/commit/42a2ae568b954190db942377df32c7fc5b89686e))
```

Comment:
```suggestion
* Ignore resources with non-http scheme ([#419](https://github.com/aws-observability/aws-rum-web/issues/419)) ([42a2ae5](https://github.com/aws-observability/aws-rum-web/commit/42a2ae568b954190db942377df32c7fc5b89686e))
```

Reply:
Ideally, yes. I updated the change log text after receiving [this feedback](https://github.com/aws-observability/aws-rum-web/pull/421#discussion_r1247165267), which suggests that ignore()'s customizability should also be communicated. 

As a compromise, I will put the commit text first, and then add the additional details after.

Reply:
updated

Reply:
Oh I see; sorry I didn't notice that.

> this description describes the default behavior of the new feature

This is by design. In my opinion, the new (default) behavior is the critical information that consumers of this package need, and the fact that it is configurable is less important.

Reply:
> In my opinion, the new (default) behavior is the critical information that consumers of this package need, and the fact that it is configurable is less important.

I think if the primary objective of the change was the default behavior, then I agree. If however, the primary objective of the change also involves giving consumers this configuration option as well, we should include it so consumers are aware of this new functionality available to them.